src/Layout/Flex/Flex.test.tsx

```diff
@@ -0,0 +1,59 @@
+import * as React from 'react';
+import '@testing-library/jest-dom/extend-expect';
+import { render } from '@testing-library/react';
+
+import { Flex } from './index';
+
+describe('<Flex /> spacing properties', () => {
+  it('should match snapshot when margin is set', () => {
+    const { asFragment } = render(<Flex m={64} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when all margin directions are set', () => {
+    const { asFragment } = render(<Flex mt={64} mr={64} mb={64} ml={64} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when padding is set', () => {
+    const { asFragment } = render(<Flex p={64} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when all padding directions are set', () => {
+    const { asFragment } = render(<Flex pt={64} pr={64} pb={64} pl={64} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when responsive margin is set', () => {
+    const { asFragment } = render(<Flex m={{ default: 32, desktopS: 64 }} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when responsive padding is set', () => {
+    const { asFragment } = render(<Flex p={{ default: 32, desktopS: 64 }} />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+});
+
+describe('<Flex /> flex properties', () => {
+  it('should match snapshot when flexDirection is set', () => {
+    const { asFragment } = render(<Flex flexDirection="column" />);
+    expect(asFragment()).toMatchSnapshot();
+  });
+
+  it('should match snapshot when justifyContent is set', () => {
```

Comment:
@rosslo , here we can test for all options? 
Like have all options in an array and match snapshot for each? Same for flexWrap, alignItems